src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,6 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Missing header comment. Header comment can be used to describe the purpose of this interface, i.e. what does it represent? Recall from tutorial that an interface is a behavioural contract.

So in this case perhaps `Represents a printable entity.....`

Reply:
So something like this?
/**
 * Represents a printable entity for all data in the addressBook.
 * Implementations should guarantee: public String getPrintableString() methods must exist and must return a String.
 */

Reply:
@lincredibleJC Yes although the second bullet point is excessive, since concrete classes that implements interfaces *must* define all methods states in that interface. Good effort improving on the commit =).